### ISSUE TITLE
Crystal 0.36.0 compile issues

### ISSUE THREAD

**Issue by bararchy** [cloudamqp/amqp-client.cr#23]
```
In amqp-client/src/amqp-client/connection.cr:3:1

 3 | require "logger"
     ^
Error: can't find file 'logger'

If you're trying to require a shard:
- Did you remember to run `shards install`?
- Did you make sure you're running the compiler in the same directory as your shard.yml?
```

**Comment by carlhoerberg** [comment]
Should've be fixed in 0.6.2 already

**Comment by carlhoerberg** [comment]
So you should be good after a `shards update`. let use know otherwise
